src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java

```diff
@@ -10,17 +10,17 @@
 
 public class InputEqualsAvoidNullTextBlocks {
     public void equalsAvoid(String myString) {
-        if (myString.equals("stuff")) { // violation
+        if (myString.equals("stuff")) { // violation 'String literal expressions should be on the left side of an equals comparison.'
         }
-        if (myString.equals("""
-                stuff""")) { // violation
+        if (myString.equals(""" // violation 'String literal expressions should be on the left side of an equals comparison.'
```

Comment:
Can't place comment here. What to do?

Reply:
Place it on line above, put this case in issue where we will support comments previous lines or we will keep this case a bit isolated and keep old style of tests on it

Reply:
done